ThingIFSDK/ThingIFSDK/ThingIFAPI+Push.swift

```diff
@@ -69,29 +69,31 @@ extension ThingIFAPI {
        installationID property is used.
      */
     open func uninstallPush(
-        _ installationID:String?,
+        _ installationID: String? = nil,
         completionHandler: @escaping (ThingIFError?)-> Void
         )
     {
-        let idParam = installationID != nil ? installationID : self.installationID
-        let requestURL = "\(baseURL)/api/apps/\(appID)/installations/\(idParam!)"
+        guard let installationID = installationID ?? self.installationID else {
+            completionHandler(ThingIFError.unsupportedError)
```

Comment:
@yongpingchen 

Suprising! Only `Aggregation` uses `ThingIFError.invalidArgument`!

Thank you for catching this!

I will fix this and will fix others in other PR.